scipy/stats/_qmc.py

```diff
@@ -379,7 +427,7 @@ def primes_from_2_to(n):
     return np.r_[2, 3, ((3 * np.nonzero(sieve)[0][1:] + 1) | 1)]
 
 
-def n_primes(n):
+def n_primes(n: int):
```

Comment:
```suggestion
def n_primes(n: int) -> List[int]:
```
The return type is really the important part here.